controllers/array_action/array_mediator_svc.py

```diff
@@ -1505,6 +1520,22 @@ def _is_vdisk_has_fcmaps(self, vdisk_uid):
         cli_volume = self._get_cli_volume_by_wwn(vdisk_uid, not_exist_err=False)
         return cli_volume and cli_volume.FC_id
 
+    def _mkhost(self, host_name, connectivity_type, ports):
+        cli_kwargs = build_create_host_kwargs(host_name, connectivity_type, ports)
+        try:
+            self.client.svctask.mkhost(**cli_kwargs)
+        except (svc_errors.CommandExecutionError, CLIFailureError) as ex:
+            if OBJ_ALREADY_EXIST in ex.my_message:
+                raise array_errors.HostAlreadyExists(host_name, self.endpoint)
+            raise ex
+
+    def _get_connectivity_type_by_initiators(self, initiators):
```

Comment:
could you move [`_mkhost`](https://github.com/IBM/ibm-block-csi-driver/blob/7d01f353bba0e617ecda732ec02ff922b99a6497/controllers/array_action/array_mediator_svc.py#L1523) and [`_get_connectivity_type_by_initiators`](https://github.com/IBM/ibm-block-csi-driver/blob/7d01f353bba0e617ecda732ec02ff922b99a6497/controllers/array_action/array_mediator_svc.py#L1532) function after [`validate_space_efficiency_matches_source`](https://github.com/IBM/ibm-block-csi-driver/blob/7d01f353bba0e617ecda732ec02ff922b99a6497/controllers/array_action/array_mediator_svc.py#L1539) function? because only `create_host` uses those functions

Reply:
no `validate_space_efficiency_matches_source` anymore..